{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -1,31 +1,25 @@
-import os
-
-import dj_database_url
 {% if cookiecutter.use_graphql == 'y' %}
 from datetime import timedelta
 {% endif %}
+import os
 
-
-def _env_get_required(setting_name):
-    """Get the value of an environment variable and assert that it is set."""
-    setting = os.environ.get(setting_name)
-    assert setting not in {None, ""}, "{0} must be defined as an environment variable.".format(setting_name)
-    return setting
+import dj_database_url
+from decouple import config
 
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-ENVIRONMENT = os.environ.get("ENVIRONMENT", "development")
+ENVIRONMENT = config("ENVIRONMENT", default="development")
 IN_DEV = ENVIRONMENT == "development"
 IN_STAGING = ENVIRONMENT == "staging"
 IN_PROD = ENVIRONMENT == "production"
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = _env_get_required("SECRET_KEY")
+SECRET_KEY = config("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = _env_get_required("DEBUG") == "True"
+DEBUG = config("DEBUG", default=False, cast=bool)
```

Comment:
@whusterj untested but this is what the change looks like. (FYI)

Reply:
This looks great! I'd be in favor of switching everything over to this.

Reply:
Tested and confirmed all this works (with one small typo bug found)